src/components/calcite-block-section/calcite-block-section.scss

```diff
@@ -7,3 +7,9 @@
   padding: 0 var(--calcite-app-side-spacing-half) var(--calcite-app-cap-spacing-half);
   animation: calcite-app-fade-in var(--calcite-app-animation-time) var(--calcite-app-easing-function);
 }
+
+.toggle--switch {
+  display: flex;
+  justify-content: space-between;
+  user-select: none;
```

Comment:
nitpick: should `user-select:none` be built into the switch? Maybe an issue for them?

Reply:
This is for the label wrapping the switch.

Reply:
Is this part of the requirements or is it a workaround for something? Not sure why disabling selection on this text is beneficial.

Reply:
I disabled this mostly for consistency. The non-switch mode doesn't allow selection. cc @asangma